src/test/simple_spin.cc

```diff
@@ -46,8 +46,7 @@ TEST(SimpleSpin, Test1)
   // Should also work with pass-by-reference:
   // (Note that we don't care about cross-threading here as-such.)
   counter = 0;
-  uint32_t urhs = 0;
-  ASSERT_EQ(counter, urhs);
+  ASSERT_EQ(0u, counter);
```

Comment:
The order of the args to ASSERT_EQ throughout the file is now inconsistent. I would make them all consistent (correct order) and, as previously mentioned, I would probably drop this particular line because it is unnecessary.